### PR TITLE
POL-321-Add-Savings-Azure-Unused-IPs

### PR DESCRIPTION
**Pull request by KVivek03**
### Description

POL-321-Add-Savings-Azure-Unused-IPs- Added savings for Azure Unused IPs

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/unused_ip_addresses/README.md [hunk 1]
```diff
@@ -2,18 +2,19 @@
 
 ## What it does  
 
-This Policy Template scans all IP addresses in the given account and identifies any unused IP addresses. If any are found, an incident report will show the IP addresses, and related information, and an email will be sent to the user-specified email address. If the user approves that the IP addresses should be deleted, the policy will delete the IP addresses. Optionally, the user can specify one or more tags that if found on an IP address will exclude the IP address from the list.
+This Policy Template scans all IP addresses in the given account and identifies any unused IP addresses. If any are found, an incident report will show the IP addresses, and related information along with the estimated monthly savings for each resource, and an email will be sent to the user-specified email address. If the user approves that the IP addresses should be deleted, the policy will delete the IP addresses. Optionally, the user can specify one or more tags that if found on an IP address will exclude the IP address from the list.
+The email will also contain Total Estimated Monthly Savings in the incident message details.
```
Comment (cdwilhelm) [inline cost/azure/unused_ip_addresses/README.md:6]: Please see the policy savings detail in the jira Epic.

File: cost/azure/unused_ip_addresses/azure_unused_ip_addresses.pt [hunk 2]
```diff
@@ -193,6 +423,10 @@ policy "pol_unused_ip_addresses" do
       field "resourceGroupName" do
         label "Resource Group Name"
       end
+      field "savings" do
+        label "Estimated Monthly Savings"
+        format "currency"
```
Comment (cdwilhelm) [inline cost/azure/unused_ip_addresses/azure_unused_ip_addresses.pt:428]: there isn't a format for currency.  it should be removed as to not confuse the reader.

Comment (douglaswth) [reply cost/azure/unused_ip_addresses/azure_unused_ip_addresses.pt:428]: Perhaps there should be a ticket for the Policy Parser to validate that the format is acceptable?

File: cost/azure/unused_ip_addresses/README.md (hunk 1 above)
Comment (KVivek03) [reply cost/azure/unused_ip_addresses/README.md]: updated the policy

File: cost/azure/unused_ip_addresses/azure_unused_ip_addresses.pt (hunk 2 above)
Comment (KVivek03) [reply cost/azure/unused_ip_addresses/azure_unused_ip_addresses.pt]: removed the currency format

**Review by cdwilhelm — APPROVED** [review]

**Pull request merged**